reana_job_controller/kubernetes_job_manager.py

```diff
@@ -1,5 +1,3 @@
-# -*- coding: utf-8 -*-
-#
```

Comment:
Why do you remove this?

Reply:
https://stackoverflow.com/a/14083123

Reply:
I have been removing those headers in couple of my previous merged PRs.

Reply:
Has this been discussed within the team at some point?
- If it hasn't, it would be better to agree on some action, like removing these lines when editing a file as you've done, or even better, run a script to remove them in bulk and assign the changes to `reana-team`.
- If it has, I probably missed it, my apologies.

cc/ @tiborsimko

Reply:
It was not discussed within the team. There were some comments between me and @tiborsimko in some PR (cannot remember which one) some time ago. So I guess an apology from my side. I don't know how to organize such changes. For me, they are very small so it didn't come to my mind to communicate them. In general, would be good to have a mechanism on how to approach such situations.